### ISSUE TITLE
Sketch Measure did not export font-family correctly.

### ISSUE THREAD

**Issue by levibostian** [utom/sketch-measure#277]
A designer I am working with on a website created a Sketch mockup of the website and exported via Sketch Measure. 

The Sketch mockup included custom fonts in it. Fonts that he has installed on his machine locally via Fontbook. 

When he exported via Sketch Measure and handed me the export, I opened it up on my machine and all of the fonts were either San Francisco or Apple System fonts. The designer, however, created custom fonts in Sketch. When he opens the Sketch Measure export on his computer, the export shows his custom fonts but when I open it, I do not see the fonts. 

Long story short, I built the website thinking that the site was built using SF and Apple System fonts so that is all I used to build it all. After I was done, he told me I had the wrong font. 

Expected behavior: Sketch measure cannot show me the fonts, sure, but the "font-family" values should be set to the font that he actually set in the Sketch file. It's ok if the UI visual I see does not have the font showing, but I should be able to tell as the developer what fonts are being used, agree? 

Does Sketch measure take care of this but I do not know how to do it?

**Comment by utom** [comment]
I will check it, thank you so much!

**Comment by siddharthksuri** [comment]
Old issue, but different variation that I encounter frequently & confuses my Dev team-
[Arial](https://www.cssfontstack.com/Arial) font family appears as [Arial Bold](https://www.cssfontstack.com/Arial-Bold) (for bold weight), within the exported view.

The latter font has serif & looks completely different from the mockups.
